sdk/communication/azure-communication-callautomation/src/test/java/com/azure/communication/callautomation/CallAutomationAsyncClientUnitTests.java

```diff
@@ -85,6 +85,7 @@ public void answerCallWithResponse() {
         assertNotNull(answerCallResult);
         assertEquals(200, answerCallResult.getStatusCode());
         assertNotNull(answerCallResult.getValue());
+        assertEquals("mediaSubscriptionId", answerCallResult.getValue().getCallConnectionProperties().getMediaSubscriptionId());
```

Comment:
are all the tests returning a Media_Subscription_Id regardless of the mediaStreamingConfiguration was provided or not?